Aberrus/Locales/deDE.lua

```diff
@@ -1,106 +1,106 @@
+	L.dread_rift = "Riss" -- Singular Dread Rift
 end
 
 L = BigWigs:NewBossLocale("The Amalgamation Chamber", "deDE")
 if L then
-	--L.custom_on_fade_out_bars = "Fade out stage 1 bars"
-	--L.custom_on_fade_out_bars_desc = "Fade out bars which belong to the boss that is out of range in stage 1."
-
-	--L.coalescing_void = "Run Away"
-	--L.shadow_convergence = "Orbs"
-	--L.molten_eruption = "Soaks"
-	--L.swirling_flame = "Tornadoes"
-	--L.gloom_conflagration = "Meteor + Run Away"
-	--L.blistering_twilight = "Bombs + Tornadoes"
-	--L.convergent_eruption = "Soaks + Orbs"
-	--L.shadowflame_burst = "Frontal Cone"
-
-	--L.shadow_and_flame = "Mythic Debuffs"
+	L.custom_on_fade_out_bars = "Phase 1 Leisten abblenden"
+	L.custom_on_fade_out_bars_desc = "Leisten, welche zum außer Reichweite befindlichen Boss in Phase 1 gehören abblenden."
+
+	L.coalescing_void = "Weglaufen"
+	L.shadow_convergence = "Kugeln"
+	L.molten_eruption = "Soaks"
+	L.swirling_flame = "Tornados"
+	L.gloom_conflagration = "Meteor + Weglaufen"
+	L.blistering_twilight = "Bombs + Tornados"
+	L.convergent_eruption = "Soaks + Kugeln"
+	L.shadowflame_burst = "Frontaler Kegel"
+
+	L.shadow_and_flame = "Mythische Debuffs"
 end
 
 L = BigWigs:NewBossLocale("The Forgotten Experiments", "deDE")
 if L then
-	--L.rending_charge_single = "First Charge"
-	--L.massive_slam = "Frontal Cone"
-	--L.unstable_essence_new = "New Bomb"
-	--L.custom_on_unstable_essence_high = "High Stacks Unstable Essence Say Messages"
-	--L.custom_on_unstable_essence_high_desc = "Say messages with the amount of stacks for your Unstable Essence debuff when they are high enough."
-	--L.volatile_spew = "Dodges"
-	--L.volatile_eruption = "Eruption"
-	--L.temporal_anomaly = "Heal Orb"
-	--L.temporal_anomaly_knocked = "Heal Orb Knocked"
+	L.rending_charge_single = "Erster Ansturm"
+	L.massive_slam = "Frontaler Kegel"
+	L.unstable_essence_new = "Neue Bombe"
+	L.custom_on_unstable_essence_high = "Sprechblasen bei hohen Stapeln von Instabiler Essenz"
+	L.custom_on_unstable_essence_high_desc = "Chatnachrichten mit der Anzahl von Stapeln Deiner Instabilen Essenz wenn diese hoch genug ist."
+	L.volatile_spew = "Ausweichen"
+	L.volatile_eruption = "Eruption"
+	L.temporal_anomaly = "Heilkugel"
+	L.temporal_anomaly_knocked = "Heilkugel weggestoßen"
 end
 
 L = BigWigs:NewBossLocale("Assault of the Zaqali", "deDE")
 if L then
 	-- These are in-game emotes and need to match the text shown in-game
 	-- You should also replace the comment (--) with the full emote as it shows in-game
-	--L.zaqali_aide_north_emote_trigger = "northern battlement" -- Commanders ascend the northern battlement!
-	--L.zaqali_aide_south_emote_trigger = "southern battlement" -- Commanders ascend the southern battlement!
+	L.zaqali_aide_north_emote_trigger = "nördliche Festungsmauer" -- Commanders ascend the northern battlement!
+	L.zaqali_aide_south_emote_trigger = "südliche Festungsmauer" -- Commanders ascend the southern battlement!
```

Comment:
We want the full ingame emote in the comment here

Reply:
Done ;)

Reply:
No, in the comments, not in the translation itself

Reply:
Okay.